documentation/airport-format.md

```diff
@@ -1,309 +1,228 @@
-where latitude and longitude are numbers that follow this format:
-    <degrees>[d|°][<minutes>m[<seconds>s]]
-
-Examples of acceptable positions:
-  [  40.94684722   ,  -76.61727778   ], // decimal degrees
-  [ "N40.94684722" , "W76.61727778"  ], // decimal degrees
-  [  "N40d56.811"  ,  "W076d37.037   ], // degrees, decimal minutes
-  [ "N40d56m48.65" , "W076d37m02.20" ]  // degrees, minutes, decimal seconds
-
-If you use a `["lat", "lon"]` combination for any `position`, you
-_must_ set the `position` of the airport as well; if you use `end`,
-`length` and `angle` are not needed (but can be used to override
-`end`).
-
-Aircraft scheduling
--------------------
-The 'type' key in an arrival or departure block determines the algorithm
-that will be used to spawn the aircraft. Each type take slightly different
-parameters in order for you to shape the airport's traffic to your liking.
-
-
-## Bare Minimum Elements
+## Comments
 
-At the very least, an arrival stream MUST have definitions for the
-following parameters. Additional may be required if the spawn method
-is set to one other than 'random'.
+### Property Descriptions
 
-            BARE MINIMUM PARAMETERS:
-   PARAMETER   REQ      PARAMETER DESCRIPTION
-+-------------+---+-------------------------------+
-| 'airlines'  | * | weighted array of airlines    |
-+-------------+---+-------------------------------+
-| 'altitude'  | * | altitude to spawn at          |
-+-------------+---+-------------------------------+
-| 'frequency' | * | spawn rate, aircraft per hour |
-+-------------+---+-------------------------------+
-| 'heading'   | * | heading to fly on spawn       |
-|    (OR)     |   |                               |
-|  'fixes'    | * | array of fixes to go to       |
-|    (OR)     |   |                               |
-|  'route'    | * | properly formatted route*     |
-+-------------+---+-------------------------------+
-| 'speed'     | * | speed to spawn at (knots)     |
-+-------------+---+-------------------------------+
-  *see index.md for route format (ex: 'BSR.BSR1.KSFO')
+#### Latitude, Longitude, Elevation
 
+For `lat, lon, elev` values, these formats are acceptable:
+    [40.94684722, -76.61727778, "866ft"],
+    ["N40.94684722", "W76.61727778", "866ft"],
+    ["N40d56.811", "W076d37.037", "866ft"],
+    ["N40d56m48.65", "W076d37m02.20", "866ft"]
 
-### Random (default)
+#### ICAO and IATA codes
 
-If the 'type' key is omitted, this spawning method will be used. The
-aircraft will be spawned at random intervals that average out to achieve
-the prescribed spawn rate. Thus, you may randomly get some back-to-back,
-and some massive gaps in your traffic, just as it is most likely to occur
-in real life.
+ICAO codes _must_ be 2-6 characters long, in uppercase letters only.
+IATA codes _must_ be 2-3 characters long, in uppercase letters only.
 
-       PARAMETERS SPECIFIC TO 'RANDOM':
```

Comment:
@erikquinn it seems a shame to throw these away, they do proved some good information.  Perhaps we could, if only temporarily, make a `SpawnPatternMethodReadme.md` file to store these for now?  We can then reference that doc from the `spawnPatternReadme.md`